presto-orc/src/main/java/com/facebook/presto/orc/reader/StreamReader.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.reader;
+
+import com.facebook.presto.orc.metadata.ColumnEncoding;
+import com.facebook.presto.orc.stream.InputStreamSources;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface StreamReader
```

Comment:
I can see how this interface is useful for the abstractions in this PR, but it's not clear from the methods on the interface how this is used for reading streams.  Basically I wonder if there's a better name for this interface.

Reply:
@tdcmeehan @highker Do you have a suggestion?